packages/list-view/lib/helper.js

```diff
@@ -1,41 +1,20 @@
 import EmberListView from './list_view';
 import EmberVirtualListView from './virtual_list_view';
 
-function createHelper (view, options) {
-  var hash = options.hash;
-  var types = options.hashTypes;
-
+export function EmberList (params, hash, options, env) {
   hash.content = hash.items;
   delete hash.items;
 
-  types.content = types.items;
-  delete types.items;
-
-  if (!hash.content) {
-    hash.content = 'this';
-    types.content = 'ID';
-  }
-
-  for (var prop in hash) {
-    if (/-/.test(prop)) {
-      var camelized = Ember.String.camelize(prop);
-      hash[camelized] = hash[prop];
```

Comment:
It's not necessary to do most of what old helper was doing to arguments. Please, read my comment above.